client/src/components/list-container/list-item-component/__tests__/ListItemComponent.test.jsx

```diff
@@ -0,0 +1,23 @@
+import renderer from "react-test-renderer";
+import listItemComponent from "../ListItemComponent";
+
+describe("render snapshots", () => {
+	it("renders correctly", () => {
+		const tree = renderer.create(<listItemComponent />).toJSON();
```

Comment:
not sure about the .JSON because the snapshots you ended with are very weird they look like the actual .jsx tags with `<listItemComponent..`.

a real snapshot result should be the end pure html with divs, spans, buttons, inputs, classes etc. 
The test itself where you `expect(tree).toMatchSnapshot();` is correct but the snapshot itself is not a good snapshot, see solution when we publish it to understand